content/doc/tutorials/_run-jenkins-in-docker.adoc

```diff
@@ -0,0 +1,125 @@
+////
```

Comment:
Think about how we could use this include in the main installing and then have additional content in the main doc.  
Or also consider putting this include next to the installing.adoc.  
Or some way to let people know these need to be kept in sync.

Reply:
I've made a note of this and will follow this up later.

Reply:
Fyi - https://issues.jenkins-ci.org/browse/WEBSITE-436